cloud/scope/route_table_reconciler.go

```diff
@@ -38,15 +38,6 @@ func (s *ClusterScope) ReconcileRouteTable(ctx context.Context) error {
 		if routeTable != nil {
 			routeTableOCID := routeTable.Id
 			s.setRTStatus(routeTableOCID, rt)
-			if s.IsTagsEqual(routeTable.FreeformTags, routeTable.DefinedTags) {
-				s.Logger.Info("route table found. No reconciliation needed", "routeTable", routeTableOCID)
```

Comment:
NIT: Should we keep this line like we do at line 49 in `cloud/scope/drg_reconciler.go`? I like the idea of the log messages saying the resource was at least found.